StortSpelProjekt/Game/src/Gamefiles/EnemyFactory.cpp

```diff
@@ -418,7 +420,30 @@ void EnemyFactory::Update(double dt)
 			m_EnemiesToSpawn--;
 			m_EnemySlotsLeft--;
 		}
+
+		// After a certain time, check if enemies are outside the map
+		// If they are, remove them
+		if (static_cast<int>(m_TotalTime) % 10 == 0)
+		{
+			m_TotalTime += 1.0f;
```

Comment:
Use a float insttead to avoid calculations with modulo. In that case, you dont have to add +1.0 aswell. Otherwise, great work :)